src/Symfony/Component/Debug/Tests/DebugClassLoaderTest.php

```diff
@@ -249,6 +271,8 @@ public function findFile($class)
             eval('namespace Test\\'.__NAMESPACE__.'; class DeprecatedParentClass extends \\'.__NAMESPACE__.'\Fixtures\DeprecatedClass {}');
         } elseif ('Test\\'.__NAMESPACE__.'\DeprecatedInterfaceClass' === $class) {
             eval('namespace Test\\'.__NAMESPACE__.'; class DeprecatedInterfaceClass implements \\'.__NAMESPACE__.'\Fixtures\DeprecatedInterface {}');
+        } elseif ('Test\\'.__NAMESPACE__.'\Float' === $class) {
+            eval('namespace Test\\'.__NAMESPACE__.'; class Float {}');
```

Comment:
It seems that we [cannot execute this on PHP 7](https://travis-ci.org/symfony/symfony/jobs/60078716#L491).